src/OpenSage.Game/Gui/Wnd/Controls/Control.cs

```diff
@@ -480,6 +485,17 @@ public static IEnumerable<Control> GetSelfAndDescendants(Control baseControl, bo
                 }
             }
         }
+
+        //TODO: check if this can be made more efficient
+        private static void CloseOpenComboboxes(Control sender)
```

Comment:
We _could_ have a per-window cache of open comboboxes, but I don't see this being a bottleneck any time soon.